### PR TITLE
truncate appVersion to 63 characters

### PR DESCRIPTION
**Pull request by bilalbokharee**

### CODE REVIEW

**Comment by stakater-user** [comment]
@bilalbokharee Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/c4ba62296c7be493a37a8e9d8562fa7dceade05d/checks) has Failed!

File: application/templates/_helpers.tpl [hunk 1]
```diff
@@ -12,7 +12,7 @@ app: {{ template "application.name" . }}
 
 {{- define "application.labels.stakater" -}}
 {{ template "application.labels.selector" . }}
-appVersion: "{{ .Values.deployment.image.tag }}"
+appVersion: "{{ .Values.deployment.image.tag | trunc 63 | trimSuffix "-" -}}"
```
Comment (rasheedamir) [inline application/templates/_helpers.tpl:15]: why truncate at 63 chars?

**Comment by stakater-user** [comment]
@bilalbokharee Validation Successful

File: application/templates/ingress.yaml [hunk 2]
```diff
@@ -23,6 +23,7 @@ spec:
       http:
         paths:
           - path: /
+            pathType: Prefix
```
Comment (rasheedamir) [inline application/templates/ingress.yaml:26]: @bilalbokharee why Prefix?

Comment (hanzala1234) [inline application/templates/ingress.yaml:26]: it can't be hardcoded. take it from values.yaml

File: application/templates/_helpers.tpl (hunk 1 above)
Comment (bilalbokharee) [reply application/templates/_helpers.tpl:15]: because some Kubernetes name fields are limited to this.

**Review by hanzala1234 — APPROVED** [review]

**Pull request merged**